jormungandr-lib/src/interfaces/mod.rs

```diff
@@ -44,7 +44,10 @@ pub use self::committee::CommitteeIdDef;
 pub use self::config::*;
 pub use self::fragment::FragmentDef;
 pub use self::fragment_log::{FragmentLog, FragmentOrigin, FragmentStatus};
-pub use self::fragment_log_persistent::PersistentFragmentLog;
+pub use self::fragment_log_persistent::{
+    get_fragments_log_files_path, load_fragments_from_folder_path, read_entries_from_files_path,
```

Comment:
I think we need more descriptive names here, they don't say anything about fragments logs actually.

Reply:
Makes sense, also there is one unneeded export too.